internal/plugins/golang/v2/init.go

```diff
@@ -44,64 +45,16 @@ func (p *initPlugin) Run() error {
 		return err
 	}
 
-	// Update the scaffolded Makefile with operator-sdk recipes.
-	// TODO: rewrite this when plugins phase 2 is implemented.
-	if err := initUpdateMakefile("Makefile"); err != nil {
```

Comment:
Call `p.run()` here instead of at the end.

Reply:
what is the diff?

Reply:
It shows no make difference, however, done. 👍